examples/slider/css/slider-1.css

```diff
@@ -0,0 +1,143 @@
+/* CSS Document */
+
+.color-picker-sliders label {
+  display: block;
+}
+
+.color-box-sliders .label {
+  display: block;
+  margin-top: 0;
+  margin-bottom: 1 em;
+}
+
+.color-picker-sliders .color-box {
+  width: 200px;
+  height: 200px;
+  border: black solid medium;
+  text-align: center;
+  padding: 0.25em;
+}
+
```

Comment:
Suggest adding 
.color-box {
  forced-color-adjust: none;
}
so the color box renders the actual color even in high contrast mode.